### ISSUE TITLE
Add  contributor's table in readme

### ISSUE THREAD

**Issue by himanshu007-creator** [rajpratyush/Social-Distancing-Detector#22]
**Is your documentation issue related to a problem? Please describe.**
----
- yes, contributor's table is missing

**Describe the solution you'd like**
----
- I will create one

**Describe alternatives you've considered**
----
-  i will add link to their contributions as well

**Additional context**
----
- Please add <code>DWOC</code> and <code>Level-2</code> tags

**Comment by rajpratyush** [comment]
there is already a contributors table existing

**Comment by himanshu007-creator** [comment]
> there is already a contributors table existing

yes, but i was suggesting a square one, that will have details of commits made by each contributor.
